File: Arcade/Intro/Ruby/fileNaming.rb
def fileNaming(names)
    set = []
    names.size.times do |i|
        aux, k = names[i], 1
        while set.include? aux
            aux = names[i]+"("+k.to_s+")"
            k += 1
        end
       set.push aux
    end
    
    return set
end

File: Arcade/Intro/javascript/fileNaming.js
function fileNaming(names) {
    let set = [];
    for(let i in names) {
        let aux = names[i];
        for(let k = 1; set.includes(aux); k++) aux = names[i]+"("+k+")";
        set.push(aux);
    }
    return set;
}